state_change_block/spec.json

```diff
@@ -0,0 +1,21 @@
+{
+    "Description": "Maintains a **state** and when **state** changes, a signal is notified that contains the `state` and `prev_state`.\nPersistence is used to maintain the **state**. If `use_persistence` is enabled, that value will populate the initial state and _not_ the `initial_state` configuration value. Any new states (from new groups) will use the configured `initial_state` for their initial state though",
+    "Properties": {
+        "backup_interval (seconds=600)": " Interval at which state is saved to disk.",
```

Comment:
Not exactly, this arg list would get close though. https://github.com/nioinnovation/nio/blob/master/nio/properties/base.py#L21

I guess I was more saying we may want to look at having the values of the property object be objects themselves, not just strings.